tests/checkpoint/test_checkpoint_with_fluent_datasources.py

```diff
@@ -1,1639 +0,0 @@
-from __future__ import annotations
-
-import copy
-import logging
-import pickle
-from typing import TYPE_CHECKING, Optional
```

Comment:
REWRITE - integrations around Spark, SQL, and Pandas should be rewritten (but I think they are covered in other files)